libkbfs/kbfs_ops.go

```diff
@@ -247,6 +246,12 @@ func (fs *KBFSOpsStandard) GetOrCreateRootNode(
 			return nil, EntryInfo{}, err
 		}
 	}
+
+	if !create && md.data.Dir.Type != Dir {
+		kbpki := fs.config.KBPKI()
+		return nil, EntryInfo{}, identifyHandle(ctx, kbpki, kbpki, h)
```

Comment:
I don't think we need an `identifyHandle` call here.  The user has not yet needed to trust any data in the folder. Was there a specific reason for the identify?